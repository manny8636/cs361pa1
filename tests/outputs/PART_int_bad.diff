--- outputs/PART_int_bad.txt	2022-02-14 17:19:49.988423244 -0500
+++ expected/PART_int_bad.txt	2022-02-13 17:48:24.374294593 -0500
@@ -0,0 +1,2 @@
+SYNTAX ERROR: 'j' is not a valid digit
+Parsing ../data/int_bad.txt failed
