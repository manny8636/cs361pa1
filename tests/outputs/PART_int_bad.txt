<<<<<<< HEAD
Parsing ../data/int_bad.txt failed
=======
>>>>>>> 149606adc8b3f0cb79ed9f7c590375c37f62be46
