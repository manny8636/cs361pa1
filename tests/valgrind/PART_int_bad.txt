==372053== Memcheck, a memory error detector
==372053== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372053== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372053== Command: ../json -i ../data/int_bad.txt
==372053== 
==372053== 
==372053== HEAP SUMMARY:
==372053==     in use at exit: 2,064 bytes in 1 blocks
==372053==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372053== 
==372053== LEAK SUMMARY:
==372053==    definitely lost: 2,064 bytes in 1 blocks
==372053==    indirectly lost: 0 bytes in 0 blocks
==372053==      possibly lost: 0 bytes in 0 blocks
==372053==    still reachable: 0 bytes in 0 blocks
==372053==         suppressed: 0 bytes in 0 blocks
==372053== Rerun with --leak-check=full to see details of leaked memory
==372053== 
==372053== For lists of detected and suppressed errors, rerun with: -s
==372053== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
