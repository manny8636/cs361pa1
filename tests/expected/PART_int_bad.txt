SYNTAX ERROR: 'j' is not a valid digit
Parsing ../data/int_bad.txt failed
